teamA/lib/Views/dashboard.dart

```diff
@@ -0,0 +1,279 @@
+import 'package:flutter/material.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+final ColorScheme customColorScheme = ColorScheme(
+  primary: Color(0xFF6A5A99), // Purple color as primary
+  onPrimary: Color(0xFFFFFFFF), // White text on primary
+  primaryContainer: Color(0xFFEDE6FF), // Light lavender
+  onPrimaryContainer: Color(0xFF2D004A), // Dark purple
+  secondary: Color(0xFF6A5A99), // Using similar purple as secondary
+  onSecondary: Colors.white, // White on secondary
+  background: Color(0xFFF5F5F5), // Light grey background
+  onBackground: Colors.black, // Black text on background
+  surface: Color(0xFFFFFFFF), // White surface color
+  onSurface: Colors.black, // Black text on surface
+  error: Colors.red, // Error color red
+  onError: Colors.white, // White on error
+  brightness: Brightness.light, // Light mode
+);
+
+class MyApp extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      debugShowCheckedModeBanner: true, // Keeps the debug banner for now
+      home: TeacherDashboard(),
+      theme: ThemeData(
+        colorScheme: customColorScheme, // Apply the custom color scheme
+        useMaterial3: true, // If you want to use Material 3 design
+      ),
+      title: 'Learning Lens',
+    );
+  }
+}
+
+class TeacherDashboard extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Learning Lens'),
+        backgroundColor: Theme.of(context)
+            .colorScheme
+            .primaryContainer, // Use primary container color
+        actions: [
+          IconButton(
+            icon: Icon(Icons.account_circle,
+                color: Theme.of(context).colorScheme.onPrimaryContainer),
+            onPressed: () {
+              // Handle profile/account actions here
+            },
+          ),
+        ],
+      ),
+      backgroundColor: Theme.of(context)
+          .colorScheme
+          .background, // Use background color from scheme
+      body: LayoutBuilder(
+        builder: (context, constraints) {
+          if (constraints.maxWidth > 600) {
```

Comment:
The interior components are wider than this. Either make them smaller, set the buttons to shrink like they do for mobile, or raise the transition cap.

Reply:
noted